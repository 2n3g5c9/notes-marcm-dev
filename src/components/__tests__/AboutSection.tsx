import React from 'react'
import renderer from 'react-test-renderer'

import { AboutSection } from '@components'

describe('AboutSection', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<AboutSection />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
