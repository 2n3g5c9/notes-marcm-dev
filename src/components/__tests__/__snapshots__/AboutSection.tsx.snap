// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutSection renders correctly 1`] = `
<div
  className="max-w-full mx-auto relative overflow-hidden"
>
  <div
    className="max-w-7xl mx-auto my-6 sm:my-8 px-8 flex items-center justify-between"
  >
    <div
      className="flex-1 min-w-0"
    >
      <h2
        className="text-2xl font-bold leading-7 text-primary sm:text-3xl sm:truncate"
      >
        Hi there ðŸ––
      </h2>
    </div>
    <div
      className="flex"
    >
      <a
        className="m-auto inline-flex items-center px-3 sm:px-4 py-1 sm:py-2 border border-transparent rounded-md shadow-sm text-sm font-medium text-tertiary bg-accent hover:bg-accentHover"
        href="mailto:"
      >
        Get in touch
      </a>
    </div>
  </div>
</div>
`;
